examples/cms-sanity/studio/package.json

```diff
@@ -8,7 +8,7 @@
     "deploy": "sanity deploy",
     "prestart": "npm run env",
     "predeploy": "npm run env",
-    "env": "cp ../.env .env.development || cp ../.env.local .env.development"
+    "env": "node -e \"const fs=require('fs');fs.existsSync('../.env')?fs.copyFileSync('../.env'), '.env.development'):fs.copyFileSync('../.env.local','.env.development');\""
```

Comment:
Feels like this is prone to errors, could we make it a JS file instead that is invoked here?

Reply:
Refactored! Apologies for the delay, forgot about the PR : /